packages/vite/src/node/plugins/worker.ts

```diff
@@ -276,12 +283,6 @@ export function webWorkerPlugin(config: ResolvedConfig): Plugin {
         }`,
         map: { mappings: '' } // Empty sourcemap to supress Rolup warning
       }
-    },
-
-    renderChunk(code) {
-      if (isWorker && code.includes('import.meta.url')) {
-        return code.replace('import.meta.url', 'self.location.href')
-      }
```

Comment:
If I'm following the conversation, isn't this a breaking change? I thought we would be keeping this for now and use MagicString for the replacement.